zerver/lib/test_runner.py

```diff
@@ -204,6 +205,9 @@ def run_subsuite(args: SubsuiteArgs) -> Tuple[int, Any]:
     # TestResult are passed TestCase as the first argument but
     # addInstrumentation does not need it.
     process_instrumented_calls(partial(result.addInstrumentation, None))
+
+    global _subsuite_index_to_pid
```

Comment:
A `global` declaration isn’t needed for assigning to `_subsuite_index_to_pid[subsuite_index]` (only for assigning to `_subsuite_index_to_pid` itself).